website/source/redirects.txt

```diff
@@ -49,6 +49,7 @@
 /docs/guides/bootstrapping.html            /docs/install/bootstrapping.html
 /docs/guides/sentinel.html                 /docs/agent/sentinel.html
 /docs/connect/proxies/sidecar-service.html /docs/connect/registration/sidecar-service.html
+/docs/enterprise/connect-multi-datacenter/index.html /docs/enterprise/index.html
```

Comment:
Will this just leave old link going somewhere confusing? Do we need a placeholder page that is not in the menu bar but just explains that replication etc. are now OSS and that advanced federation etc. are enterprise or something?

I can see why it is a little awkward to maintain that given the "not calling it multi-dc anymore" stance and probably not enough links here to care that much but just a thought?

Reply:
I went with the "probably not enough links here to care that much" – I figured we didn't need to special-case it in any particular way. I considered redirecting to the connect internals page but that felt confusing as well.